app/views/balances/show.html.haml

```diff
@@ -0,0 +1,100 @@
+- if @balance
+  - if browser.device.mobile?
```

Comment:
Is it practical to ask for device type? what if user just narrows down the window? Which alternatives are available to build responsive design?